pywren_ibm_cloud/storage/__init__.py

```diff
@@ -0,0 +1,198 @@
+import os
```

Comment:
@JosepSampe is it good design practice to keep actually code functions in __init__ ? I think this class only used for some init short statements or imports, not really code ...

Reply:
@gilv It can be moved outside

Reply:
@JosepSampe I think __init__ should contain very limited code, so if it's not an issue for you, can you then move it outside of __init__? otherwise, we can fix it with later pull requests..

Reply:
@gilv Done